seed.js

```diff
@@ -0,0 +1,15 @@
+const data = require('./seedData.js');
+const db = require('./db/index.js');
+
+const randomProduct = (obj) => {
+    var keys = Object.keys(obj);
+    
+    // only adding 2 products for now, will change to 100 later
+    for (let i = 1; i <= 2; i++) {
+        var randomInt = Math.floor(Math.random() * keys.length);
+        var randomItem = keys[randomInt];
+        db.insertOne(randomItem, JSON.stringify(obj[randomItem]));
```

Comment:
If you decide to change up your insertOne method of your DB per my other comment, you would need to change up the arguments you are passing into it here.